code/shared-mobility/shared-mobility-smrt.js

```diff
@@ -72,3 +72,23 @@ function smallButton(id, view_id, url, fa_icon, button_label, is_disabled = fals
 
   if (callback) callback();
 }
+$(document).on("knack-scene-render.scene_257", function (event, page) {
+  // update iframe src from detail field
+  var iframe_url = $('a[href*="webappviewer"]').attr("href");
+  $(".view_534").hide();
```

Comment:
Was testing done to make sure the Details views are being hidden? I use a pound sign instead of period to hide my views. For example: "#view_960"

Cool to know if both are valid and if they operate any differently within the handler.